create_mountaineer_app/create_mountaineer_app/templates/project/pyproject.toml

```diff
@@ -9,8 +9,8 @@ authors = ["{{author_name}} <{{author_email}}>"]
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.11"
-
+python = "^3.10"
+toml = "^0.10.2"
```

Comment:
Per above comment, you can conditionally include toml just for certain python versions by doing something like:

```toml
toml = { version = "^0.10.2", python = "<=3.10" }
```